### PR TITLE
fix #19

### LINKED ISSUE #19: Error on executing the script
```
 default: /tmp/vagrant-shell: line 5: /vagrant/provision/resources/core/memcached-admin/provision.sh: Permission denied
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```
I got that running the script, I think that is a problem of another repo anyway.

### PR DESCRIPTION
**Pull request by Mte90**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -2,13 +2,33 @@
...
+current_directory=`pwd`
+
 echo -e "\033[0;32mHello! This script will generate a build folder containing a copy of VVV with instructions and installers.\033[0m"
 echo -e "\033[0;32mOnce this script finishes succesfully, you can copy the contents of the build folder on to USB drives!\033[0m"
 echo ""
 echo -e "\033[0;33mPrior warning, this script takes a while to run, don't be surprised if it's 1 hour+\033[0m"
-echo -e "\033[0;33mAlso, do not run this on a USB drive. Run it on an SSD then copy the final result\033[0m"
+for path in "${not_suggested_path[@]}" ; do
+    if [[ $current_directory = *$path* ]]; then
+        read -p "You are trying run this on a USB drive (slowness and issue on executing the script). Do you want to proceed?" -n 1 -r
```
Comment (tomjn) [inline run.sh]: I think it should recommend that this be ran on an SSD/HD still, else people are likely to just say yes

**Review by tomjn — CHANGES_REQUESTED** [review]

File: run.sh (hunk 1 above)
Comment (Mte90) [reply run.sh]: so only change the text

**Review by tomjn — APPROVED** [review]

**Comment by tomjn** [comment]
Hopefully we can integrate this into the vagrant file too and run it when contributor day mode is enabled

**Comment by Mte90** [comment]
Not a bad idea to port also there :-)

**Pull request merged**